tests/integrations/test_dsm.py

```diff
@@ -2,7 +2,7 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2023 Datadog, Inc.
 
-from utils import weblog, interfaces, scenarios, released
+from utils import weblog, interfaces, scenarios, released, context
 from utils.tools import logger
 
 
```

Comment:
So far, the test is not activated. You must replace the `?` declared here for nodejs.

Reply:
we were thinking we could add it after we do a release on dsm

Reply:
As you wish.